### DISCUSSION TITLE
Charge limit problem

### DISCUSSION THREAD

**Discussion by MC-dusk** [seerge/g-helper#1745] [General]
ASUS TUF Gaming A15 FA507XV_FA507XV
I set the charge limit at 80% and it works well. But when I set it between 85% and 95% (including), it will charge to 100%.
Setting below 50% (including) will also cause this.
Any idea?

---

0.137.0 ghelper
312 bios
ASUS System Control Interface v3 3.1.20.0

**Comment by seerge** [comment]
Some devices just don't support anything except 80%

>[ Battery charge limiter is not working](https://github.com/seerge/g-helper?tab=readme-ov-file#battery-charge-limiter-is-not-working)
> It could be that Asus services are overwriting this limit after. You may want to stop them by clicking "Stop" in the Asus Services section (under Extra). Please note: For some devices not every charge limit % may be working. Try to set standard 80% to be sure.

**Reply by MC-dusk** [reply]
Thank you, I've read it before and finally forget. I think that's why I try to set it at 80% at first, ha..
